config/core/test-configs.yaml

```diff
@@ -2186,6 +2204,7 @@ test_configs:
   - device_type: mt8173-elm-hana
     test_plans:
       - baseline
+      - baseline-cip-nfs
```

Comment:
This one is failing but it's a known issue, the network is broken in mainline at the moment (well, missing something in defconfig):
https://staging.kernelci.org/test/plan/id/6185115274f9828eaaa6c483/

So it's not a problem for this PR.

Reply:
OK: https://staging.kernelci.org/test/plan/id/6188ddfdbc4b48580ea6c45f/

(staging-mainline has some extra patches to fix this platform, it won't work with current mainline)